drivers/hopwa_caper/app/models/hopwa_caper/enrollment.rb

```diff
@@ -0,0 +1,130 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HopwaCaper
+  class Enrollment < GrdaWarehouseBase
+    include SqlHelper
+    self.table_name = 'hopwa_caper_enrollments'
+
+    has_many :services, class_name: 'HopwaCaper::Service', foreign_key: [:hud_enrollment_id, :data_source_id, :report_instance_id], primary_key: [:hud_enrollment_id, :data_source_id, :report_instance_id]
+
+    def self.as_report_members
+      all.map do |record|
+        ::HudReports::UniverseMember.new(
+          universe_membership_type: sti_name,
+          universe_membership_id: record.id,
+        )
+      end
+    end
+
+    scope :overlapping_range, ->(start_date:, end_date:) {
+      where('entry_date <= :end_date AND (exit_date >= :start_date OR exit_date IS NULL)', start_date: start_date, end_date: end_date)
```

Comment:
We've worked really hard to keep raw SQL out of the code base unless absolutely necessary.  This is a pattern we have in many classes already, can we use arel for this? 
Cribbed from service history enrollment:
```
scope :overlapping_range, ->(start_date:, end_date:) do
  at = arel_table  
  where(at[:exit_date].gteq(start_date).or(at[:exit_date].eq(nil)).and(at[:entry_date].lteq(end_date)))
end
```

Reply:
yes, I'll make a pass back through and transform any stray sql strings. I find it much easier to work with strings than arel so I tend to start there and transform them before I hand it off